tests/mobly/base_test_test.py

```diff
@@ -2215,7 +2216,9 @@ def test_something(self):
           continue
         hit = True
         self.assertEqual(c['a'], content['a'])
+        self.assertIn('timestamp', c)
         self.assertIsNotNone(c['timestamp'])
```

Comment:
This line would fail with an IndexError instead of an AssertionError if timestamp were missing, which is why I added the line above.